creator-node/src/services/sync/skippedCIDsRetryService.js

```diff
@@ -17,41 +18,20 @@ class SkippedCIDsRetryQueue {
     if (!nodeConfig || !libs) {
       throw new Error(`${LogPrefix} Cannot start without nodeConfig, libs`)
     }
-
-    this.queue = new Bull('skipped-cids-retry-queue', {
-      redis: {
-        port: nodeConfig.get('redisPort'),
-        host: nodeConfig.get('redisHost')
-      },
+    this.nodeConfig = nodeConfig
+    this.libs = libs
```

Comment:
can you just make sure that this queue specifically is working as expected

Reply:
also am i correct that only changes here are just moving from constructor to init?

Reply:
yeah that's correct just moved some things because they should be awaited and the constructor isn't async. I verified that I see the queue running and I see the success log sometimes able to save more than 0 retried CIDs, so that's good